patches/redirectToDVLogin.js

```diff
@@ -0,0 +1,28 @@
+function redirectToLoginPage() {
+	window.location.pathname = `/${window.location.pathname.split("/")[1]}/LoginEndpoint.aspx`;
+}
+
+window.appendModule({
+    onlyOnReloads: true,
+    run: redirectToLoginPage,
+    doesRunHere: () =>
+        window.location.hostname === "dziennik-uczen.vulcan.net.pl"
+		&& !window.location.pathname.split("/")[2]
+});
+
+const pathSegment = window.location.pathname.split("/")[1];
+
+const newSpanContent = `
```

Comment:
I think we should change it dynamically based on previous content instead of overwriting it, it would prevent problems like password url getting changed or this getting outdated in any other way